app/controllers/concerns/blacklight/search_context.rb

```diff
@@ -111,9 +111,7 @@ def setup_next_and_previous_documents
       response, documents = search_service.previous_and_next_documents_for_search index, search_state.reset(current_search_session.query_params).to_hash
 
       search_session['total'] = response.total
-      @search_context_response = response
-      @previous_document = documents.first
```

Comment:
I guess we should backport some deprecation for this?

Reply:
I suppose we could make these deprecated instance variables in a backport.

Reply:
👍